src/components/container/AppDownload/AppDownload.tsx

```diff
@@ -0,0 +1,66 @@
+﻿import React, { useState } from 'react'
+import './AppDownload.css'
+import MyDataHelps, { ProjectInfo } from '@careevolution/mydatahelps-js'
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import PlatformSpecificContent from '../PlatformSpecificContent/PlatformSpecificContent';
+import { useInitializeView } from '../../../helpers/Initialization';
+import Card from '../../presentational/Card';
+import TextBlock from '../../presentational/TextBlock';
+import googlePlayDownload from '../../../assets/google-play-download.svg';
+import appStoreDownload from '../../../assets/app-store-download.svg';
+import language from "../../../helpers/language";
+import LocalizedString from "../../presentational/LocalizedString";
+
+export interface AppDownloadProps {
+	previewProjectPlatforms?: string[]
+	previewDevicePlatform?: string;
+}
+
+export default function (props: AppDownloadProps) {
+	const [projectInfo, setProjectInfo] = useState<ProjectInfo>();
+
+	useInitializeView(() => {
+		if (props.previewProjectPlatforms) {
+			// @ts-ignore
+			setProjectInfo({name: 'PROJECT', platforms: props.previewProjectPlatforms || []});
+			return;
+		}
+
+		MyDataHelps.getProjectInfo().then(function (projectInfo) {
+			setProjectInfo(projectInfo);
+		});
+	});
+
+	if (!projectInfo) {
+		return null;
+	}
+
+	return (
+		<div className="mdhui-app-download">
+			<PlatformSpecificContent platforms={['Web']} previewDevicePlatform={props.previewDevicePlatform}>
+				<Card>
+					<div className="mdhui-app-download-title">
+						<LocalizedString stringName="app-download-title"/>
```

Comment:
Would someone want to customize this title? (not sure how that works w/ Localization)

Reply:
Possibly.  I think if we provide that capability, localization would be the responsibility of the overriding application.

I'm not sure if any/many of our current reusable components allow for that kind of customization. If we want to be more flexible about that, maybe we can do something at a global level that won't require modifying each of the individual components to support it.